### PR TITLE
updated elasticsearch to 6.6.1

### PR DESCRIPTION
**Pull request by monotek**
**What type of PR is this?**
> Uncomment only one, leave it on its own line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
/kind flake

**What this PR does / why we need it**:
* updates elasticsearch to 6.6.1

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by desaintmartin** [comment]
Hehe, should we even go to 6.6.0 now?

**Comment by monotek** [comment]
Yes, it's already on my todo list :-)

**Comment by monotek** [comment]
@desaintmartin 
6.6.0 image is used now. 
also in https://github.com/kubernetes/kubernetes/pull/71251

File: cluster/addons/fluentd-elasticsearch/es-image/Dockerfile [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM docker.elastic.co/elasticsearch/elasticsearch-oss:6.3.2
+FROM docker.elastic.co/elasticsearch/elasticsearch-oss:6.5.3
```
Comment (coffeepac) [inline cluster/addons/fluentd-elasticsearch/es-image/Dockerfile]: Please use es 6.6.0 as mentioned in notes.

File: cluster/addons/fluentd-elasticsearch/es-image/Makefile [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 PREFIX = staging-k8s.gcr.io
 IMAGE = elasticsearch
-TAG = v6.3.0
+TAG = v6.5.3
```
Comment (coffeepac) [inline cluster/addons/fluentd-elasticsearch/es-image/Makefile]: Again, 6.6.0.

File: cluster/addons/fluentd-elasticsearch/es-statefulset.yaml [hunk 3]
```diff
@@ -63,17 +63,17 @@ spec:
...
       labels:
         k8s-app: elasticsearch-logging
-        version: v6.3.0
+        version: v6.6.0
         kubernetes.io/cluster-service: "true"
     spec:
       serviceAccountName: elasticsearch-logging
       containers:
-      - image: k8s.gcr.io/elasticsearch:v6.3.0
+      - image: k8s.gcr.io/elasticsearch:v6.6.0
```
Comment (coffeepac) [inline cluster/addons/fluentd-elasticsearch/es-statefulset.yaml]: Please rebase against master to get latest location.

**Review by coffeepac — CHANGES_REQUESTED** [review]
Changes requested are entirely my fault for dragging so much.

Please try and build the 6.6.0 images after rebasing.  You should be able to.  If not, please ping me on slack.

File: cluster/addons/fluentd-elasticsearch/es-statefulset.yaml [hunk 4]
```diff
@@ -63,17 +63,17 @@ spec:
...
       labels:
         k8s-app: elasticsearch-logging
-        version: v6.3.0
+        version: v6.6.1
         kubernetes.io/cluster-service: "true"
     spec:
       serviceAccountName: elasticsearch-logging
       containers:
-      - image: k8s.gcr.io/elasticsearch:v6.3.0
+      - image: k8s.gcr.io/elasticsearch:v6.6.1
```
Comment (coffeepac) [inline cluster/addons/fluentd-elasticsearch/es-statefulset.yaml]: this is the old location.  we're not planning on updating this repository any more.  the image should be at:  `gcr.io/fluentd-elasticsearch/elasticsearch`

Comment (coffeepac) [reply cluster/addons/fluentd-elasticsearch/es-statefulset.yaml]: nuts, the other PR only changed the default build location, not what the image repository.  

Do you want to make those changes here or I can push another PR.  Also need to move the fluentd image.

Comment (monotek) [reply cluster/addons/fluentd-elasticsearch/es-statefulset.yaml]: changed image to gcr.io/fluentd-elasticsearch/elasticsearch

**Pull request merged**